variables.tf

```diff
@@ -322,6 +322,16 @@ variable "create_jump_public_ip" {
   default     = true
 }
 
+variable "jump_public_ip_allocation_method" {
+  default = "Static"
+  type    = string
+}
```

Comment:
Could you please add description for all the new variables introduced in this PR?

Reply:
Done!